tools/fuzzer/platform/imx8-host.c

```diff
@@ -266,7 +266,7 @@ static int imx8_irq_handler(int irq, void *context)
 
 	/* Interrupt arrived, check src */
 	status = imx_mu_read(fuzzer, IMX8_MU13_A_BAR, IMX_MU_xSR);
-
+	status &= 0xffffffffL;
```

Comment:
Lets add a comment like this:
```
/* MU has 32bit registers so only keep interesting bits */
 	status &= 0xffffffffL;
```

Reply:
Removed this commit following the discussion here - https://github.com/thesofproject/sof/pull/3239#discussion_r460707127